Ginger/GingerAutoPilot/APIModelLib/SwaggerApi/OpenApiBase.cs

```diff
@@ -306,13 +345,50 @@ public ObservableList<OptionalValue> GetListOfParamEnums(SwaggerParameter swagge
             return lstOptions;
         }
 
-        public void SetOptionalValue(ObservableList<AppModelParameter> AppModelParameters, Dictionary<string, string> listExampleValues)
+        public static Dictionary<string, string> ExampleValueDict(SwaggerOperation operations)
+        {
+
+            Dictionary<string, string> exampleValues = new Dictionary<string, string>();
+            
+            try
+            {
+                if (operations.RequestBody.Content.ElementAt(0).Value.Examples != null && operations.RequestBody != null)
```

Comment:
Add operations.RequestBody != null this condition before the first condition

Reply:
added